apps/examples/testcase/ta_tc/task_manager/utc/utc_task_manager_main.c

```diff
@@ -46,7 +46,7 @@ static int *addr2;
 static void test_handler(int sig, tm_msg_t *info)
 {
 	int ret = strncmp(info->si_value.sival_ptr, TM_SAMPLE_MSG, strlen(TM_SAMPLE_MSG));
-	TC_ASSERT_EQ("task_manager_unicast_p", ret, 0);
+	TC_ASSERT_EQ("task_manager_unicast", ret, 0);
```

Comment:
strncmp

Reply:
if assert here, it is not from strncmp, but task_manager_unicast.
So I will not fix this.